### ISSUE TITLE
Rules warning on non-define calls

### ISSUE THREAD

**Issue by cvisco** [cvisco/eslint-plugin-requirejs#23]
Rules have been sometimes warning on non-define calls. For example, in `no-object-define`:

``` js
// This warns, as it should
define('name', {});

// This also warns, and it shouldn't:
foo('name', {});
```

This applies to all current rules.
